components/bl-granim-component/component.json

```diff
@@ -0,0 +1,358 @@
+{
+  "id": "c_035c0cdcf668f2a3ea71c991caea3e59",
+  "name": "Granim",
+  "description": "The Granim component allows the user to create fluid and interactive gradient animations.",
+  "showInToolbox": true,
+  "faIcon": "paint-roller",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
+      "name": "direction",
+      "label": "Direction",
+      "showInSettings": true,
+      "defaultValue": "left-right",
+      "hasLogicHandler": false,
+      "handlerId": "directionLogic",
+      "options": [
+        {
+          "value": "left-right",
+          "label": "left-right"
+        },
+        {
+          "value": "diagonal",
+          "label": "diagonal"
+        },
+        {
+          "value": "top-bottom",
+          "label": "top-bottom"
+        },
+        {
+          "value": "radial",
+          "label": "radial"
+        },
+        {
+          "value": "custom",
+          "label": "custom"
+        }
+      ],
+      "settingTooltip": "The orientation of the gradient"
+    },
+    {
+      "type": "text",
+      "name": "x0",
+      "label": "x0",
+      "settingTooltip": "Starting x-coordinate for custom direction",
+      "showInSettings": true,
+      "defaultValue": "",
+      "hasLogicHandler": false,
+      "handlerId": "x0Logic"
+    },
+    {
+      "type": "text",
+      "name": "y0",
+      "label": "y0",
+      "showInSettings": true,
+      "defaultValue": "",
+      "hasLogicHandler": false,
+      "handlerId": "y0Logic",
+      "settingTooltip": "Starting y-coordinate for custom direction"
+    },
+    {
+      "type": "text",
+      "name": "x1",
+      "label": "x1",
+      "showInSettings": true,
+      "defaultValue": "",
+      "hasLogicHandler": false,
+      "handlerId": "x1Logic",
+      "settingTooltip": "End x-coordinate for custom direction"
+    },
+    {
+      "type": "text",
+      "name": "y1",
+      "label": "y1",
+      "showInSettings": true,
+      "defaultValue": "",
+      "hasLogicHandler": false,
+      "handlerId": "y1Logic",
+      "settingTooltip": "End y-coordinate for custom direction"
+    },
+    {
+      "type": "json",
+      "name": "colors",
+      "label": "Colors",
+      "settingTooltip": "Set the colors for the gradient",
+      "showInSettings": true,
+      "defaultValue": "[]",
+      "hasLogicHandler": false,
```

Comment:
That should have an logicHandler, and defaultValue

Reply:
@Fialkaksu pay attention for that

Reply:
The component no longer has this parameter.